src/widgets/scrollbar.rs

```diff
@@ -315,22 +331,23 @@ impl<'a> Scrollbar<'a> {
     /// └─────────── begin
     /// ```
     ///
-    /// Only sets begin_symbol, end_symbol and track_symbol if they already contain a value.
+    /// Only sets `begin_symbol`, `end_symbol` and `track_symbol` if they already contain a value.
     /// If they were set to `None` explicitly, this function will respect that choice. Use their
     /// respective setters to change their value.
     ///
     /// This is a fluent setter method which must be chained or used as it consumes self
+    #[allow(clippy::needless_pass_by_value)] // Breaking change
```

Comment:
Hey, this is a lint in the pedantic list, which doesn't trigger unless you enable pedantic or this lint itself. Was there some other rationale for adding this line?

https://rust-lang.github.io/rust-clippy/master/#/needless_pass_by_value

Reply:
This is one of the many pedantic lints (probably except one or two) that should be enabled for all libraries in my opinion. I ran over this file with pedantic, nursery and a bunch of restricted lints to improve it. Afterwards I removed the `#![warn(…)]` block in the top of the file, but it looks like I kept the individual allow for this one.

Reply:
Ah got it - makes sense - thanks for the attention to detail :)